app/src/main/java/com/github/se/gomeet/ui/navigation/BottomNavigationMenu.kt

```diff
@@ -33,7 +33,7 @@ fun BottomNavigationMenu(
 ) {
   NavigationBar(
       modifier = Modifier.navigationBarsPadding().height(80.dp),
-      containerColor = Color.Transparent,
+      containerColor = NavBarUnselected,
```

Comment:
I personally feel like the transparent colour fits better (on phones with a home bar)